src/processFrames.js

```diff
@@ -4,40 +4,105 @@ const _ = require('lodash');
 
 const Errors = require('./Errors');
 
+const AR_CALCULATION_PRECISION = 0.01;
+
+const SQUARE_AR_COEFFICIENT = 1;
+const SQUARE_AR = '1:1';
+
+const TRADITIONAL_TV_AR_COEFFICIENT = 1.333;
+const TRADITIONAL_TV_AR = '4:3';
+
+const HD_VIDEO_AR_COEFFICIENT = 1.777;
+const HD_VIDEO_AR = '16:9';
+
+const UNIVISIUM_AR_COEFFICIENT = 2;
+const UNIVISIUM_AR = '18:9';
+
+const WIDESCREEN_AR_COEFFICIENT = 2.33;
+const WIDESCREEN_AR = '21:9';
+
 function processFrames(frames) {
     if (!Array.isArray(frames)) {
         throw new TypeError('process method is supposed to accept an array of frames.');
     }
 
+    const audioFrames            = processFrames.filterAudioFrames(frames);
```

Comment:
You don't use `audioFrames` array, so you may replace `filterAudioFrames` function with smth like `hasAudioFrames` and use [`Array.some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) method. It's much more efficient.

Reply:
good remark